modules/sr/robot/ruggeduino_devices.py

```diff
@@ -72,16 +72,18 @@ def __init__(
         webot: Robot,
         device_name: str,
         limiter: OutputFrequencyLimiter,
+        pin_num: int,
     ) -> None:
         self._name = device_name
```

Comment:
I think `_name` is no longer used -- could we drop it?

Reply:
Indeed `device_name` is only used to get the sensor object at initialisation so there is no need to store it.

Reply:
Corrected in 03ccbc09bd43e5a3818d31d6632f2a9961d7ee51